doc/conf.py

```diff
@@ -338,22 +338,25 @@
 		('boards/arm/nrf51_ble400/doc/nrf51_ble400', 'boards/arm/nrf51_ble400/doc/index'),
 		('boards/arm/nrf51_blenano/doc/nrf51_blenano', 'boards/arm/nrf51_blenano/doc/index'),
 		('boards/arm/nrf51_pca10028/doc/nrf51_pca10028', 'boards/arm/nrf51dk_nrf51422/doc/index'),
-		('boards/arm/nrf51_pca10028/doc/nrf51_pca10028/index', 'boards/arm/nrf51dk_nrf51422/doc/index'),
+		('boards/arm/nrf51_pca10028/doc/index', 'boards/arm/nrf51dk_nrf51422/doc/index'),
+		('boards/arm/nrf51_pca10031/doc/index', 'boards/arm/nrf51dongle_nrf51422/doc/index'),
 		('boards/arm/nrf51_vbluno51/doc/nrf51_vbluno51', 'boards/arm/nrf51_vbluno51/doc/index'),
 		('boards/arm/nrf52810_pca10040/doc/nrf52810_pca10040', 'boards/arm/nrf52dk_nrf52810/doc/index'),
-		('boards/arm/nrf52810_pca10040/doc/nrf52810_pca10040/index', 'boards/arm/nrf52dk_nrf52810/doc/index'),
+		('boards/arm/nrf52810_pca10040/doc/index', 'boards/arm/nrf52dk_nrf52810/doc/index'),
+		('boards/arm/nrf52811_pca10056/doc/index', 'boards/arm/nrf52840dk_nrf52811/doc/index'),
+		('boards/arm/nrf52833_pca10100/doc/index', 'boards/arm/nrf52833dk_nrf52833/doc/index'),
 		('boards/arm/nrf52840_pca10056/doc/nrf52840_pca10056', 'boards/arm/nrf52840dk_nrf52840/doc/index'),
-		('boards/arm/nrf52840_pca10056/doc/nrf52840_pca10056/index', 'boards/arm/nrf52840dk_nrf52840/doc/index'),
-		('boards/arm/nrf9160_pca10090/doc/nrf9160_pca10090/index', 'boards/arm/nrf9160dk_nrf9160/doc/index'),
-		('boards/arm/nrf52840_pca10090/doc/nrf52840_pca10090/index', 'boards/arm/nrf9160dk_nrf52840/doc/index'),
+		('boards/arm/nrf52840_pca10056/doc/index', 'boards/arm/nrf52840dk_nrf52840/doc/index'),
+		('boards/arm/nrf9160_pca10090/doc/index', 'boards/arm/nrf9160dk_nrf9160/doc/index'),
+		('boards/arm/nrf52840_pca10090/doc//index', 'boards/arm/nrf9160dk_nrf52840/doc/index'),
```

Comment:
Double slash in the "new style old name" URL. This creates an invalid redirection. Same story for nrf52840_pca10059 below.

Reply:
Thanks, fixed!